sklearn/metrics/tests/test_classification.py

```diff
@@ -469,7 +469,7 @@ def test_precision_recall_f1_score_multiclass_pos_label_none():
     # compute scores with default labels introspection
     p, r, f, s = precision_recall_fscore_support(y_true, y_pred,
                                                  pos_label=None,
-                                                 average='weighted')
+                                                 average='binary')
```

Comment:
This one is intentionally testing the case where `precision_recall_fscore_support` should treat a binary problem as if it's multiclass. So this should be weighted or macro, not binary.